docs/roadmap.md

```diff
@@ -30,6 +30,12 @@ KubeSphere Roadmap demonstrates a list of open source product development plans
 
 #### DevOps
 
+- [ ] Create / Edit Pipeline Process Optimization.
+- [ ] S2I/B2I supports webhook.
+- [ ] Image registry optimization.
```

Comment:
Image registry management optimization, provide control panel for private image registry (e.g. manage images within Harbor)